lighthouse-core/test/audits/third-party-summary-test.js

```diff
@@ -37,16 +37,55 @@ describe('Third party summary', () => {
         mainThreadTime: 127.15300000000003,
         blockingTime: 18.186999999999998,
         transferSize: 30827,
+        subItems: {
+          items: [
+            {
+              blockingTime: 18.186999999999998,
+              mainThreadTime: 127.15300000000003,
+              transferSize: 30827,
+              url: 'https://www.googletagmanager.com/gtm.js?id=GTM-Q5SW',
+            },
+          ],
+          type: 'subitems',
+        },
       },
       {
         entity: {
           text: 'Google Analytics',
           type: 'link',
           url: 'https://www.google.com/analytics/analytics/',
         },
-        mainThreadTime: 95.15600000000005,
+        mainThreadTime: 95.15599999999999,
         blockingTime: 0,
         transferSize: 20913,
+        subItems: {
+          items: [
+            {
+              blockingTime: 0,
+              mainThreadTime: 55.246999999999986,
+              transferSize: 12906,
+              url: 'https://www.google-analytics.com/analytics.js',
+            },
+            {
+              blockingTime: 0,
+              mainThreadTime: 5.094,
+              transferSize: 3161,
+              url: 'https://www.google-analytics.com/cx/api.js?experiment=qvpc5qIfRC2EMnbn6bbN5A',
+            },
+            {
+              blockingTime: 0,
+              mainThreadTime: 7.01,
+              transferSize: 2949,
+              url: 'https://www.google-analytics.com/cx/api.js?experiment=jdCfRmudTmy-0USnJ8xPbw',
+            },
+            {
+              blockingTime: 0,
+              transferSize: 1897,
+              url: 'lighthouse-core/audits/third-party-summary.js | otherValue # 0',
```

Comment:
```suggestion
              url: expect.any(String),
```

then add
`expect(result.details.items[1].subItems.items[3]).toBeDisplayString('Other resources')` to test what we're really after?